docs/architecture/pairing-application-instance-with-compass.md

```diff
@@ -0,0 +1,49 @@
+# Pairing Application Instance with Compass
```

Comment:
There is no such term as "Application Instance" - at least not yet. Maybe you can just omit `Instance` here, to not introduce new term.